tests/test-performance-lastpostmodified.php

```diff
@@ -103,6 +103,6 @@ public function test__override_lastpostmodified__is_set_post() {
 	public function test__override_lastpostmodified__is_not_set_post() {
 		$actual = get_lastpostmodified( 'gmt', 'post' );
 
-		$this->assertEquals( $this->post_modified_gmt, $actual );
```

Comment:
This property did not exist :-) interestingly, the test failed only with WP nightly.